src/Nerdbank.Streams.Tests/NestedStreamTests.cs

```diff
@@ -47,15 +47,21 @@ public void Slice_InputValidation()
     [Fact]
     public void Length()
     {
-        Assert.Throws<NotSupportedException>(() => this.stream.Length);
+        Assert.Equal(10, this.stream.Length);
```

Comment:
Avoid magic numbers where possible.

```suggestion
        Assert.Equal(DefaultNestedLength, this.stream.Length);
```